command/eval_list.go

```diff
@@ -0,0 +1,182 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/posener/complete"
+)
+
+type EvalListCommand struct {
+	Meta
+}
+
+func (c *EvalListCommand) Help() string {
+	helpText := `
+Usage: nomad eval list [options]
+
+  List is used to list the set of evaluations processed by Nomad.
+
+General Options:
+
+  ` + generalOptionsUsage(usageOptsDefault) + `
+
+Eval Status Options:
+
+  -verbose
+    Show full information.
+
+  -per_page
+    How many results to show per page.
+
+  -page_token
+    Where to start pagination.
+
+  -job
+    Only show evaluations for this job ID.
+
+  -status
+    Only show evaluations with this status.
+
+  -json
+    Output the evaluation in its JSON format.
+
+  -t
+    Format and display evaluation using a Go template.
+`
+
+	return strings.TrimSpace(helpText)
+}
+
+func (c *EvalListCommand) Synopsis() string {
+	return "Display evaluation status and placement failure reasons"
```

Comment:
This needs to be updated for the `nomad eval -help` text output:

```console
$ nomad eval -help
Usage: nomad eval <subcommand> [options] [args]
...
Subcommands:
    list      Display evaluation status and placement failure reasons
    status    Display evaluation status and placement failure reasons
```

Reply:
How about "List the set of evaluations processed by Nomad", so that it matches the `nomad eval list -help` text?